nixos/modules/services/printing/cupsd.nix

```diff
@@ -140,6 +141,8 @@ in
         '';
       };
 
+      cups.package = lib.mkPackageOption pkgs "cups" {};
```

Comment:
I find it a bit weird that this module's options are prefixed with `services.printing` rather than `services.cups`, which leads to this option having to be prefixed with `cups.` to make any sense. But that is a bit of a sidenote. Given the circumstances, this seems fine.

Reply:
```suggestion
      package = lib.mkPackageOption pkgs "cups" {};
```
we should just name it package

Reply:
done!